leftwm-core/src/display_action.rs

```diff
@@ -38,7 +38,7 @@ pub enum DisplayAction {
     DestroyedWindow(WindowHandle),
 
     /// Tell a window that it is to become focused.
-    WindowTakeFocus(Window),
+    WindowTakeFocus(Window, Option<WindowHandle>),
```

Comment:
Not particularly fan of the tuple variant for this variant in particular because it is not obvious at all to understand what are the fields. I'd suggest converting to a struct variant:

```suggestion
    WindowTakeFocus {
        window: Window,
        previous_handle: WindowHandle,
    },
```